docs/src/components/Playground/renderConfig.ts

```diff
@@ -17,13 +21,34 @@ export const babelConfig = {
 }
 
 export const imports = {
-  '@stardust-ui/docs-components': DocsComponent,
-  '@stardust-ui/react': Stardust,
-  classnames: Classnames,
-  lodash: _,
-  react: React,
-  'react-dom': ReactDOM,
-  'react-fela': ReactFela,
+  '@stardust-ui/docs-components': {
+    version: docsComponentsPackageJson.version,
+    module: DocsComponent,
+  },
+  '@stardust-ui/react': {
+    version: stardustReactPackageJson.version,
+    module: Stardust,
+  },
+  classnames: {
+    version: stardustReactPackageJson.dependencies['classnames'],
+    module: Classnames,
+  },
+  lodash: {
+    version: stardustReactPackageJson.devDependencies['@types/lodash'],
```

Comment:
```suggestion
    version: stardustReactPackageJson.dependencies['lodash'],
```

Oops, still points to `@types/`